doc/rst/developer/bestPractices/ContributorInfo.rst

```diff
@@ -102,91 +319,92 @@ Submit pull request
-    also pass testing for ``start_test -performance``).
+  Note: Ideally, someone should volunteer to review your pull request within a
+  day or two. If this doesn't happen, feel free to make some noise. Ideally the
+  review should take place within a few days, though timing may vary depending
+  on other deadlines.
 
-  * Changes to the Chapel implementation should not cause
-    regressions. Developers are responsible for doing a degree of testing
-    that's appropriate for their change (see guidance below) and then can rely
-    on nightly regression testing to worry about the full cross-product of
-    configurations.
+* See `Reviewer responsibilities`_ for details on what performing a review on
+  another contributor's code entails.
 
-    * At a minimum, patches should pass correctness testing for the full test/
-      directory hierarchy for:
+.. _Work with your reviewers:
 
-      * ``CHPL_*_PLATFORM=linux64``
-      * ``CHPL_*_COMPILER=gnu``
-      * ``CHPL_COMM=none``
-      * ``CHPL_TASKS=<default>``
+Work with your reviewers
+++++++++++++++++++++++++
 
-    * Most developers will start by focusing on a subdirectory of tests that
-      exercise the features they changed, or test/release/ as a suite of tests
-      that exercises a rich and important slice of the language.
-
-    * Changes that are likely to affect multi-locale executions should also be
-      tested against tests that exercise multi-locale capabilities with
-      ``CHPL_COMM=gasnet``.  A common subset is: ``test/release/``,
-      ``test/multilocale/``, and ``test/distributions/``.
-
-    * Changes that are likely to cause portability issues should be tested
-      against different platforms and compilers to avoid fallout in the nightly
-      testing to the extent possible.
+* Iterate with the reviewer until you're both satisfied. If you can't come to
+  agreement, one of you should bring other developers (individually or via
+  chapel-developers_) into the conversation to get a broader opinion (one of
+  the jobs of the reviewer is to serve as a proxy for other developers, or to
+  bring those developers into the conversation if they feel unqualified to do
+  so).
 
-  * Note that the quickest way to do testing is to use the parallel testing
-    system across a large number of workstations.  If you have limited testing
-    resources available to you, you can request that a member of the core
-    Chapel team help.
 
-.. _Test System: https://github.com/chapel-lang/chapel/blob/master/doc/rst/developer/bestPractices/TestSystem.rst
+.. _Test changes:
 
-.. _Mail chapel-developers@ for review and merge:
+Test changes
+~~~~~~~~~~~~
 
-Mail chapel-developers@ for review and merge
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+If you did not have the resources to perform at least a full correctness test
+run, this is the point at which a reviewer would do so for you.  See
+`Testing guidance`_ above for more details on what an appropriate amount of
+testing is.
```

Comment:
I think there are three things to link to from this section:
 1. The earlier section, basically saying "You already ran some testing, right"?
 2. The Policy section, saying PRs should pass at least full local testing, talking about configurations etc etc
 3. The HOWTO section or the Test System document, to describe the mechanics of running tests